base/compiler/ssair/inlining.jl

```diff
@@ -1165,6 +1223,10 @@ function assemble_inline_todo!(ir::IRCode, state::InliningState)
                 ir.stmts[idx][:inst] = quoted(calltype.val)
                 continue
             end
+            # Refuse to inline OpaqueClosures we can't see otherwise, to preserve the
```

Comment:
should this be either:
```suggestion
            # Otherwise: refuse to inline OpaqueClosures we can't see, to preserve the
```

or:
```suggestion
            # Refuse to inline OpaqueClosures we can't see, to preserve the
```

Or am i not understanding what is going on here?

Reply:
This comment seems to have gotten lost.